content/en/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -511,6 +520,15 @@ If you want to use project quotas, you should:
   mounted with project quotas enabled. For both XFS and ext4fs, the
   mount option is named `prjquota`.
 
+
+If you don't want to use project quotas, you should:
+
+* Disable the `LocalStorageCapacityIsolationFSQuotaMonitoring`
+  [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+  using the `featureGates` field in the
+  [kubelet configuration](/docs/reference/config-api/kubelet-config.v1beta1/)
+  or the `--feature-gates` command line flag.
```

Comment:
Is this right?

BTW I think `--feature-gates` is deprecated, and (nit) it's a command line argument not a flag.

Reply:
Changed this line. Please take a look.